odl/solvers/advanced/proximal_operators.py

```diff
@@ -208,17 +385,18 @@ def proximal_convexconjugate_l2(space, lam=1, g=None):
     Function for the proximal operator of the convex conjugate of the
     functional F where F is the l2-norm
 
-        F(x) =  lam 1/2 ||x - g||_2^2
+        F(x) =  lam ||x - g||_2
 
     with x and g elements in ``space``, scaling factor lam, and given data g.
 
-    The convex conjugate, F_cc, of F is given by
+    The convex conjugate, F^*, of F is given by
 
-        F_cc(y) = 1/lam (1/2 ||y/lam||_2^2 + <y/lam,g>)
+        F^*(y) = {0 if ||x-g|| < lam, infty else}
 
-    The proximal operator of F_cc is given by
+    The proximal operator of ``sigma * F^*`` where ``sigma`` is a step size
+    is given by
 
-        prox_sigma[F_cc](y) = (y - sigma g) / (1 + sigma/lam)
+        prox[sigma * F^*](y) = (y - sigma g) / (1 + sigma/lam)
```

Comment:
Yes, there I definitely buy it.

On 2016-05-21 23:34, Jonas Adler wrote:
> In odl/solvers/advanced/proximal_operators.py
> <https://github.com/odlgroup/odl/pull/401#discussion_r64140604>:
> 
>>  
>> -        prox_sigma[F_cc](y) = (y - sigma g) / (1 + sigma/lam)
>> +        prox[sigma * F^*](y) = (y - sigma g) / (1 + sigma/lam)
> 
> So thats where my confusion came from, this was obviously copy-paste.
> Thought you commented on the other place (where it should be correct).
> 
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/odlgroup/odl/pull/401/files/5a721d1d1325be17940d8eeb33e35bdd4879ad8a#r64140604>
> 

-- 

Dr Holger Kohr
KTH - Royal Institute of Technology
Dpt of Mathematics
100 44 Stockholm
Tel +46 8 790-6652
Mobil +46 72 0119332